presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -453,7 +455,29 @@ else if (x instanceof Timestamp) {
     public void addBatch()
             throws SQLException
     {
-        throw new NotImplementedException("PreparedStatement", "addBatch");
+        checkOpen();
+        batchParameters.add(ImmutableMap.copyOf(parameters));
```

Comment:
Here we could make map into a list, and verify right number and right positions of parameters were set